modules/tooltip/react/stories/stories.tsx

```diff
@@ -1,76 +1,130 @@
 /// <reference path="../../../../typings.d.ts" />
-import * as React from 'react';
-import {storiesOf} from '@storybook/react';
+/** @jsx jsx */
+import {jsx} from '@emotion/core';
+import React from 'react';
+import {CSSObject} from '@emotion/styled';
 import withReadme from 'storybook-readme/with-readme';
-import {Popper} from '@workday/canvas-kit-react-common';
+
+import {Popper, Placement} from '@workday/canvas-kit-react-common';
 import {xIcon} from '@workday/canvas-system-icons-web';
+import {Card} from '@workday/canvas-kit-react-card';
+import {IconButton, Button, DeleteButton} from '@workday/canvas-kit-react-button';
+import {Tooltip, TooltipContainer, useTooltip} from '@workday/canvas-kit-react-tooltip';
 
-import {IconButton} from '../../../button/react';
-import Tooltip from '../index';
 import README from '../README.md';
 
-interface TooltipWrapperState {
-  open: boolean;
-  anchorEl: HTMLElement | null;
-}
+export default {
+  title: 'Components|Popups/Tooltip/React',
+  component: Tooltip,
+  decorators: [withReadme(README)],
+};
 
-class TooltipWrapper extends React.Component<{}, TooltipWrapperState> {
-  private tooltipRef: React.RefObject<HTMLDivElement>;
-  public constructor(props: {}) {
-    super(props);
-    this.tooltipRef = React.createRef();
-    this.state = {
-      open: false,
-      anchorEl: null,
-    };
-  }
+export const Default = () => {
+  return (
+    <Tooltip title="Close">
+      <IconButton variant={IconButton.Variant.Circle} icon={xIcon} aria-label="Close" />
+    </Tooltip>
+  );
+};
 
-  public render() {
-    const {open} = this.state;
-    return (
-      <div>
-        <h3>Hover over the icon.</h3>
-        <div
-          style={{display: 'inline-flex'}}
-          ref={this.tooltipRef}
-          onMouseEnter={this.open}
-          onMouseLeave={this.close}
-          onFocus={this.open}
-          onBlur={this.close}
-        >
-          <IconButton
-            variant={IconButton.Variant.Circle}
-            icon={xIcon}
-            aria-labelledby={'123'}
-            aria-label="Close"
-          />
+export const DescribeType = () => {
+  return (
+    <Tooltip type="describe" title="The service will restart after this action">
+      <DeleteButton>Delete</DeleteButton>
+    </Tooltip>
+  );
+};
+
+export const CustomContent = () => {
+  return (
+    <Tooltip
+      type="describe"
+      title={
+        <div>
+          This is a <em>custom</em> tooltip with <strong>custom HTML</strong>.
+          <br />
+          Any HTML is valid here.
+          <br />
+          Keep it short and note that assistive technology will read this content as text-only and
+          maybe have verbosity turned off to the point where this content is completely ignored.
+          Consider a different way to describe important content.
```

Comment:
Can we do a less verbose custom HTML example so that we can show our true intent for this feature? I feel like this will encourage bad practice. Design tries to keep people to one line if possible. Maybe we move the messaging to below the target?

```suggestion
        <div>
          This is a <em>custom</em> tooltip with <strong>custom HTML</strong>
```